### PR TITLE
Added the light-theme in a separate classs

### PR DESCRIPTION
**Pull request by yosefbeder**

### CODE REVIEW

**Comment by yosefbeder** [comment]
I think it's very good

**Comment by M0ST4FA** [comment]
I think it's amazing too

**Comment by M0ST4FA** [comment]
shall we merge it

**Comment by yosefbeder** [comment]
No I realized some issues

File: index.css [hunk 1]
```diff
@@ -27,3 +27,4 @@ button {
   font-size: 20px;
   padding: 5px;
 }
+hi
```
Comment (yosefbeder) [inline index.css:30]: This isn't valid css

**Pull request merged**